tests/components/switch/test_command_line.py

```diff
@@ -155,3 +156,13 @@ def test_state_code(self):
 
             state = self.hass.states.get('switch.test')
             self.assertEqual(STATE_ON, state.state)
+
+
+    def test_assumed_state(self):
```

Comment:
The test seems fair.  I would just make sure it's more clear what you are testing.

If I were writing this test, the title would be:
``` test_assumed_state_should_be_true_if_command_state_is_none ```